recipe/data.go

```diff
@@ -1788,19 +1788,19 @@ var AgeingSexRatios = Response{
 	},
 }
 
-// Local authority ageing statistics - net flows for over 65s and over 85s
-var AgeingNetFlows = Response{
+// Local authority ageing statistics - net internal migration for over 65s and over 85s
+var AgeingNetInternalMigration = Response{
 	ID:     "c5875623-0874-429d-bbc6-63d2d84e6159",
-	Alias:  "Local authority ageing statistics - net flows for over 65s and over 85s",
+	Alias:  "Local authority ageing statistics - net internal migration for over 65s and over 85s",
```

Comment:
We are using `over 65s and over 85s` here, but we use `65 and over and 85 and over` above.  Potentially worth considering aligning them for consistency.

Reply:
business area made no request to change this but it does make sense to align them so i have